cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc

```diff
@@ -9,102 +9,134 @@ toc::[]
+. Select *Web* from the _Redirect URI_ dropdown and enter the value of the OAuth callback URL you retrieved in the previous step.
+. After providing the required information, click *Register* to create the application.
 +
 image:azure-portal_register-an-application-page.png[Azure Portal - Register an application page]
 +
-Then, click on the "Certificates & secrets" sub-blade and select "New client secret". Fill in the details request and make note of the generated client secret value, as you'll use it in a later step. You won't be able to retrieve it again.
+. Select the *Certificates & secrets* sub-blade and select *New client secret*.
++
+image:azure-portal_certificates-secrets-page.png[Azure Portal - Certificates and secrets page]
+
+. Complete the requested details and store the generated client secret value. This secret is required later in this process.
++
+[IMPORTANT]
+====
+After initial setup, you cannot see the client secret. If you did not record the client secret, you must generate a new one.
+====
 +
-image:azure-portal_certificates-secrets-page.png[Azure Portal - Certificates & secrets page]
 image:azure-portal_add-a-client-secret-page.png[Azure Portal - Add a Client Secret page]
-image:azure-portal_copy-client-secret-page.png[Azure Portal - Copy Client Secret page]
+
+. Select the *Overview* sub-blade and note the `Application (client) ID` and `Directory (tenant) ID`. You will need these values in a future step.
 +
-Then, click on the "Overview" sub-blade and make note of the "Application (client) ID" and "Directory (tenant) ID". You'll need those values in a later step as well.
+image:azure-portal_copy-client-secret-page.png[Azure Portal - Copy Client Secret page]
 
-[id="rosa-mobb-entra-id-configure-optional-claims"]
-== Configure optional claims
+[id="rosa-mobb-entra-id-configure-claims"]
+== Configuring the application registration in Entra ID to include optional and group claims
 
-To provide OpenShift with enough information about the user to create their account, we will configure Entra ID to provide two optional claims, specifically "email" and "preferred_username". For more information on optional claims in Entra ID, see link:https://learn.microsoft.com/en-us/azure/active-directory/develop/optional-claims[the Microsoft documentation].
+To give OpenShift enough information about the user to create their account, you configure Entra ID to give two optional claims, specifically `email` and `preferred_username`. For more information about optional claims in Entra ID, see link:https://learn.microsoft.com/en-us/azure/active-directory/develop/optional-claims[the Microsoft documentation].
 
-Click on the "Token configuration" sub-blade and select the "Add optional claim" button.
+In addition to individual user authentication, OpenShift provides group claim functionality. This functionality allows an OpenID Connect (OIDC) identity provider, such as Entra ID, to offer a user's group membership for use within OpenShift.
```

Comment:
"OpenShift" appears enough times in this PR that I think it might be intentional, but just flagging it in case it should be `{product-title}`. Applies to the whole PR.